pkgs/top-level/all-packages.nix

```diff
@@ -33430,8 +33432,8 @@ with pkgs;
 
   nerd-font-patcher = callPackage ../applications/misc/nerd-font-patcher { };
 
-  netmaker = callPackage ../applications/networking/netmaker {subPackages = ["."];};
-  netmaker-full = callPackage ../applications/networking/netmaker { };
+  netmaker = callPackage ../applications/networking/netmaker { };
+  netclient = callPackage ../applications/networking/netclient {};
```

Comment:
```suggestion

  netclient = callPackage ../applications/networking/netclient { };
```

Reply:
@wexder you'll probably still want to add the space between braces here for consistency